system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -72,17 +73,17 @@ double computeCpuActivePercentage(
 
   const double active_time = measurement2.getActiveTime() - measurement1.getActiveTime();
   const double total_time = (measurement2.getIdleTime() + measurement2.getActiveTime()) -
-    (measurement1.getIdleTime() + measurement2.getActiveTime());
+    (measurement1.getIdleTime() + measurement1.getActiveTime());
```

Comment:
unrelated

Reply:
Fixed in https://github.com/ros-tooling/system_metrics_collector/pull/18.